backend/src/main/java/se/chalmers/dat265/group1/model/dto/DigitalPartStl.java

```diff
@@ -0,0 +1,12 @@
+package se.chalmers.dat265.group1.model.dto;
+
+import se.chalmers.dat265.group1.model.DigitalPart;
+
+public class DigitalPartStl extends DigitalPart {
+    private String path;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "path" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)